google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingProductAd.java

```diff
@@ -237,11 +237,11 @@ private String addStandardShoppingCampaign(
             // the ads from immediately serving. Set to ENABLED once you've added
             // targeting and the ads are ready to serve.
             .setStatus(CampaignStatus.PAUSED)
-            // Sets the bidding strategy to Manual CPC (with eCPC enabled)
+            // Sets the bidding strategy to Manual CPC (with eCPC disabled)
             // Recommendation: Use one of the automated bidding strategies for Shopping campaigns
             // to help you optimize your advertising spend. More information can be found here:
             // https://support.google.com/google-ads/answer/6309029.
-            .setManualCpc(ManualCpc.newBuilder().setEnhancedCpcEnabled(true).build())
+            .setManualCpc(ManualCpc.newBuilder().setEnhancedCpcEnabled(false).build())
```

Comment:
Should we be setting this at all? What will happen if the user just does `.setManualCpc(ManualCpc.newBuilder())`?

Reply:
I thought about this while working on it, and observed that if we do not set enhanced_cpc_enabled explicitly, by default the value is `true` by checking the result for the created campaign's settings in the Google Ads UI. With that, I feel it might be better to encourage setting `false` explicitly in this example to avoid confusions (not setting eCPC enabled in code -> created campaign with eCPC enabled -> the actual bidding behavior is with eCPC disabled).

P.S. More info on my test: I did it with the python client lib by changing the according line to `client.copy_from(campaign.manual_cpc, client.get_type("ManualCpc"))`. I think that should equal to the Java `.setManualCpc(ManualCpc.newBuilder().build())`, by referring to related code in [this example](https://developers.google.com/google-ads/api/samples/add-complete-campaigns-using-batch-job#python).

Reply:
Thanks for checking. Given it defaults to `true`, I agree with the approach of explicitly setting it to `false`.